services/app-api/forms/wp.json

```diff
@@ -27,7 +27,7 @@
               "props": {
                 "href": "https://www.govinfo.gov/content/pkg/PLAW-109publ171/pdf/PLAW-109publ171.pdf",
                 "target": "_blank",
-                "ariaLabel": "Link opens in new tab"
+                "aria-label": "User Guide and Help File (Link opens in new tab)"
```

Comment:
This doesn't line up with the visual content 🤔 
should probably be something like
`6071(a)(1) of the Deficit Reduction Act (DRA) (Link opens in new tab)`

Reply:
oh whoops! I don't know how I mixed that up, to be honest.... 😅